packages/runtime-dom/__tests__/customElement.spec.ts

```diff
@@ -172,17 +172,19 @@ describe('defineCustomElement', () => {
 
       e.setAttribute('bar', '')
```

Comment:
In the original test cases, converting `''` to `true`, I'm not sure if this aligns with expectations, but from a boolean perspective, it should be converted to `false`.